.circleci/config.yml

```diff
@@ -347,6 +322,141 @@ jobs:
                 rm mp3d_example.zip
                 cd ..
               fi
+              if [ ! -f ./habitat-sim/data/scene_datasets/coda/coda.glb ]
+              then
+                cd habitat-sim
+                wget --no-check-certificate 'https://docs.google.com/uc?export=download&id=1Pc-J6pZzXEd8RSeLM94t3iwO8q_RQ853' -O coda.zip
+                unzip coda.zip -d data/scene_datasets
+                rm coda.zip
+                cd ..
+              fi
+      - run:
+          name: Build sim documentation
+          no_output_timeout: 20m
+          command: |
+              export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
+              . activate habitat;
+              cd habitat-sim
+              pip install -r requirements.txt --progress-bar off
+              pip install imageio imageio-ffmpeg
+              conda install -y -c conda-forge doxygen==1.8.16
+              conda install -y  jinja2 pygments docutils
+              git submodule update --init --recursive --jobs 4
+              while [ ! -f ~/cuda_installed ]; do sleep 2; done # wait for CUDA
+              sudo apt install --allow-change-held-packages \
+                  texlive-base \
+                  texlive-latex-extra \
+                  texlive-fonts-extra \
+                  texlive-fonts-recommended
+
+              # Rebuild with all options enabled
+              ./build.sh --with-cuda --with-bullet
+              cd docs
+              git submodule update --init
+              ./build-public.sh
+      - run:
+          name: Install Habitat Lab
+          command: |
+              export PATH=$HOME/miniconda/bin:$PATH
+              . activate habitat;
+              cd habitat-sim
+              #Rebuild headless
+              python setup.py install --headless --with-bullet --with-cuda
+              cd ..
+              if [ ! -d ./habitat-lab ]
+              then
+                git clone -q --depth 1 https://github.com/facebookresearch/habitat-lab.git
+              fi
+              cd habitat-lab
+              pip install -r requirements.txt --progress-bar off
+              ln -s ../habitat-sim/data data
+              touch ~/miniconda/pip_deps_installed
+      - save_cache:
+          key: conda-{{ checksum "habitat-sim/.circleci/config.yml" }}
+          background: true
+          paths:
+            - ~/miniconda
+      - run:
+          name: Run Habitat Lab tests
```

Comment:
>I also went ahead and isolate the habitat-sim testing from the habitat-lab testing so that it should take a reasonable amount of time to test each. Now we can easily run multiple tasks in parallel.

I am not sure it's done in the right way. As I understand Habitat Lab tests will use conda binary installed habitat-sim, not master. Otherwise, this job should dependent on `install_and_test_ubuntu` job.

Reply:
Noticed habitat sim rebuild commands: `./build.sh --with-cuda --with-bullet`.